<ui version="4.0" >
 <class>Form</class>
 <widget class="QWidget" name="Form" >
  <property name="windowModality" >
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>240</horstretch>
    <verstretch>162</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>500</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>500</width>
    <height>460</height>
   </size>
  </property>
  <property name="sizeIncrement" >
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="baseSize" >
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget" >
   <property name="windowModality" >
    <enum>Qt::WindowModal</enum>
   </property>
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>10</y>
     <width>491</width>
     <height>381</height>
    </rect>
   </property>
   <property name="minimumSize" >
    <size>
     <width>491</width>
     <height>381</height>
    </size>
   </property>
   <property name="currentIndex" >
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_3" >
    <attribute name="title" >
     <string>Cadastro de Produtos (Omega Sistemas)</string>
    </attribute>
    <attribute name="icon" >
     <iconset>../../../opt/kde/share/icons/crystalsvg/48x48/apps/bluefish.png</iconset>
    </attribute>
    <widget class="QLineEdit" name="edt_nome" >
     <property name="geometry" >
      <rect>
       <x>110</x>
       <y>30</y>
       <width>361</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label" >
     <property name="geometry" >
      <rect>
       <x>110</x>
       <y>12</y>
       <width>128</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text" >
      <string>Nome do Produto:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_id" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>30</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="readOnly" >
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_6" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>13</y>
       <width>21</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text" >
      <string>ID:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_codbarras" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>76</y>
       <width>261</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2" >
     <property name="geometry" >
      <rect>
       <x>10</x>
       <y>60</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text" >
      <string>Codigo de Barras:</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_novo" >
   <property name="geometry" >
    <rect>
     <x>2</x>
     <y>400</y>
     <width>68</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text" >
    <string>&amp;Novo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_atualizar" >
   <property name="geometry" >
    <rect>
     <x>138</x>
     <y>400</y>
     <width>68</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text" >
    <string>&amp;Atualizar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_excluir" >
   <property name="geometry" >
    <rect>
     <x>206</x>
     <y>400</y>
     <width>68</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text" >
    <string>&amp;Excluir</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_relatorio" >
   <property name="geometry" >
    <rect>
     <x>274</x>
     <y>400</y>
     <width>68</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text" >
    <string>&amp;Relatorio</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_sair" >
   <property name="geometry" >
    <rect>
     <x>410</x>
     <y>400</y>
     <width>68</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text" >
    <string>&amp;Sair</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_localizar" >
   <property name="geometry" >
    <rect>
     <x>342</x>
     <y>400</y>
     <width>68</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text" >
    <string>&amp;Localizar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_salvar" >
   <property name="geometry" >
    <rect>
     <x>70</x>
     <y>400</y>
     <width>68</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text" >
    <string>&amp;Salvar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
